dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -1126,24 +1165,19 @@ sources:
                   meta:
                     description: >
                       user20 (No. of Commercial Units) should not be null
-          - name: user24
-            description: '{{ doc("shared_column_card_proration_rate") }}'
-            tests:
-              - not_null:
-                  name: iasworld_dweldat_card_proration_rate_not_null
```

Comment:
`user24` is no longer the canonical proration rate column, so we remove this test. See the PR description for details on my (so far failed) attempts to recreate it on `external_propct`.

Reply:
It's probably going to require some chats with Mirella to figure out how these new columns work, we didn't need to touch them for 2024 modeling but will going forward. I opened a new issue #392 to cover this.